src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -154,6 +154,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <_DocumentationFileProduced>true</_DocumentationFileProduced>
     <_DocumentationFileProduced Condition="'$(DocumentationFile)'==''">false</_DocumentationFileProduced>
 
+    <!-- Whether or not a reference assembly is produced-->
+    <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
```

Comment:
To not _create_ them in this project. They'll still be consumed if produced in other projects.

Better names welcome.